test/CMakeLists.txt

```diff
@@ -24,8 +24,19 @@
 # 2/ Command lines for each binary
 # For each binary, one or more command lines shall be specified. Such command lines are stored as string lists
 # in variables named lstTestCLI_BINARY, with BINARY being one of the suffixes mentioned above.
-# eg. list(APPEND lstTestsCLI_Serial
-#        "../run_exodiff_test.py --executable _EXECUTABLE_ --input-deck _INPUT_DECK_"
+# eg. list(APPEND lstTestsCLI_Kokkos
+#        "../run_exodiff_test.py --executable _EXECUTABLE_ --input-deck _INPUT_DECK_ --ranks 1"
+#        "../run_exodiff_test.py --executable _EXECUTABLE_ --input-deck _INPUT_DECK_ --ranks 2"
+#        )
+#
+# Each command line will generate a new test. By default, the name of each test is suffixed by a number
+# incremented (0,1,2...) for each different CLI.
+# It is possible to OPTIONALLY specify strings of text as custom suffixes by adding an optional variable named
+# the same as the above list, with _Suffixes added at the end.
+#
+# eg. list(APPEND lstTestsCLI_Kokkos_Suffixes
```

Comment:
Good.
Can you do the same thing for the NimbleSM_MPI executable?
And also NimbleSM_Tpetra?

Reply:
Done, will submit a new PR